src/main/java/org/jenkinsci/plugins/ghprb/GhprbTrigger.java

```diff
@@ -302,7 +302,10 @@ public void run() {
         // one isn't there
         return this.job.scheduleBuild2(job.getQuietPeriod(), cause, new ParametersAction(values), findPreviousBuildForPullId(pullIdPv));
     }
-    
+
+    private String escapeText(String text) {
+        return text.replace("\n", "\\n").replace("\r", "\\r").replace("\"", "\\\"");
```

Comment:
I'm not replacing a \, I'm replacing a " with a \". It just looks weird in the Java syntax itself as these characters need to be replaced for the replacement, too. And with this replacement, our job now works again fine.